pkg/worker/handler/endpoint/handler.go

```diff
@@ -17,20 +17,18 @@ const (
 
 var (
 	mapping = map[string]map[string]string{
+		"api": {
+			"production": "https://api.splits.org/metrics",
+		},
```

Comment:
This is the exception. There is no `testing` nor `staging` env for this special case.